modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-cart-impl/src/main/java/com/liferay/headless/commerce/delivery/cart/internal/resource/v1_0/CartResourceImpl.java

```diff
@@ -336,9 +336,12 @@ private CommerceOrder _addCommerceOrder(
 				cart.getAccountId());
 		}
 
-		return _commerceOrderService.addCommerceOrder(
+		CommerceOrder commerceOrder = _commerceOrderService.addCommerceOrder(
 			commerceChannelGroupId, accountEntry.getAccountEntryId(),
 			commerceCurrencyId, _getCommerceOrderTypeId(cart));
+
+		return _commerceOrderService.updatePrintedNote(
+			commerceOrder.getCommerceOrderId(), cart.getPrintedNote());
```

Comment:
This is the wrong place to do it. When the order is being added, only the ADD_TO_CART permission needs to be considered. PrintedNote update needs to be done as part of `_updateOrder`

Reply:
@brianikim Is it alright now?